<link rel="stylesheet" href="/assets/css/styles.css">
<script src="/assets/js/document_include.js"></script> 

<h1>Introducing "met"</h1>
nvib is proud to reintroduce <code>met</code>, which has gone under a complete redesign and overall, a scientific analysis platform for metabarcoded microbiome and metagenomic data, which makes it possible to do large scale search and comparisions across tens of thousands of datasets. A long standing problem in microbiome and Environmental DNA has been the inability to compute across large number of datasets. You can read more about the first generation  of met here: <a href="[https://ecoevorxiv.org/rwnd3/](https://academic.oup.com/database/article/doi/10.1093/database/baac032/6583522)">met paper</a> Also, you can Checkout the latest <a href="/news.html">nvib news</a>, find out <a href="/team.html">who we are</a> and <a href="/contact.html">contact us</a>

<figure class="embiggen">
  <img src="/assets/met.png" alt="met ecosystem">
  <figcaption>The met ecosystem can be deployed in the public or your private cloud (<a href="/assets/met.png">png</a>)</figcaption>
</figure>

<code>met</code> can be setup in the public cloud or on your private cloud hardware to scale your microbiome and eDNA analysis. <code>met</code> is deployed via containers which means that you have a range of deployment options to choose from.

<figure class="embiggen">
  <img src="/assets/met-barcoding.png" alt="your pipeline">
  <figcaption>Simplify your metagenomic workflow with met (<a href="/assets/met-barcoding.png">png</a>)</figcaption>
</figure>

you can simplify your workflow with met, saving time with automation and increasing bioinformatic possiblities. met can take raw sequence reads and process to create more than just taxon tables, it can search all of your sequences for particular species, compare all of your samples, and scale your metabarcode analyis to previously unattainable levels.

<script>
document.include('/assets/menu.html')
</script>
